lucien-ios/AddNewComicBook/AddComicViewController.swift

```diff
@@ -0,0 +1,233 @@
+        backButton.tintColor = UIColor.black
+        navigationItem.leftBarButtonItem = backButton
+
+        // NavigationBar 'Finish' Button
+        finishButton = UIBarButtonItem(title: "Finish", style: .plain, target: self, action: nil)
+        finishButton.setTitleTextAttributes([NSAttributedStringKey.font: LucienTheme.Fonts.Muli.semiBold.getFont(size: 17) ?? UIFont()], for: .normal)
+        finishButton.tintColor = LucienTheme.finishButtonGrey
+        finishButton.isEnabled = false
+        navigationItem.rightBarButtonItem = finishButton
+    }
+
+    private func configureViewController() {
+        // Cover Photo Cell
+        configureAddCoverUIButton(button: coverPhotoButton)
+
+        // Comic Title
+        configureTextFieldBorder(textField: comicTitleTextField)
+        comicTitleTextField.attributedPlaceholder = createPlaceHolderAttributedString(textFieldPlaceholderText: "Comic Title")
+        comicTitleTextField.addTarget(self, action: #selector(AddComicViewController.comicTitleEditingChanged), for: .editingChanged)
+
+        // Story Title
+        configureTextFieldBorder(textField: storyTitleTextField)
+        storyTitleTextField.attributedPlaceholder = createPlaceHolderAttributedString(textFieldPlaceholderText: "Story Title")
+
+        // Volume
+        configureTextFieldBorder(textField: volumeTextField)
+        volumeTextField.attributedPlaceholder = createPlaceHolderAttributedString(textFieldPlaceholderText: "Volume Number")
+
+        // Issue
+        configureTextFieldBorder(textField: issueTextField)
+        issueTextField.attributedPlaceholder = createPlaceHolderAttributedString(textFieldPlaceholderText: "Issue Number")
+
+        // Publisher
+        configureTextFieldBorder(textField: publisherTextField)
+        publisherTextField.attributedPlaceholder = createPlaceHolderAttributedString(textFieldPlaceholderText: "Marvel, DC Comics, Image, Dark Horse…")
+
+        // Release Date
+        configurePickerUIButton(button: selectADateButton)
+
+        // Genre
+        configurePickerUIButton(button: selectAGenreButton)
+
+        // Condition
+        configurePickerUIButton(button: selectAConditionButton)
+    }
+
+    private func configureTextFieldBorder(textField: UITextField) {
+        let bottomBorder = CALayer()
+        bottomBorder.frame = CGRect(x: 0.0, y: textField.frame.height - 3.0, width: textField.frame.width, height: 1.0)
+        bottomBorder.backgroundColor = LucienTheme.silver.cgColor
+        textField.borderStyle = UITextBorderStyle.none
+        textField.layer.addSublayer(bottomBorder)
+    }
+
+    private func configurePickerUIButton(button: UIButton) {
+        button.setImage(UIImage(named: "dropDownArrow"), for: .normal)
+        button.semanticContentAttribute = .forceRightToLeft
+        button.imageEdgeInsets.left = button.frame.width - 24
+        button.tintColor = LucienTheme.dark
+    }
+
+    private func configureAddCoverUIButton(button: UIButton) {
```

Comment:
Unless you're reusing this function with another cover photo button, you don't need to pass in the button, you can just use `coverPhotoButton` directly